docs/how-to/index.rst

```diff
@@ -14,13 +14,39 @@ adapt the steps to fit your specific requirements.
    :maxdepth: 1
 
    Get started - quick guide <get-started>
-   Use Rockcraft's GitHub Action <rockcraft-pack-action>
-   Contribute to Rockcraft documentation <contribute-docs>
-   Build the documentation <build-docs>
-   Cut existing slices with Chisel <use-chisel>
-   Create a package slice for Chisel <create-slice>
-   Install a custom package slice <install-slice>
-   Publish a slice definition <publish-slice>
-   Convert an entrypoint to a Pebble layer <convert-to-pebble-layer>
-   Publish a rock to a registry <publish-a-rock.rst>
-   Migrate a Docker image to a chiselled rock <migrate-to-chiselled-rock.rst>
+
+Chisel
+------
+
+Chisel is a tool that lets you keep only the required files from one or more apt
+packages and discard the rest. Upgrade your Chisel slicing skills by learning
+how to cut, install and publish slices.
```

Comment:
```suggestion
Upgrade your Chisel slicing skills by learning
how to cut, install and publish slices.
```

Let's leave the explanations for the explanations page...it's less error-prone this way